### PR TITLE
APIO-261 As an APIO Architect developer, I want to have a runnable example in the Apio Architect repository

### PR DESCRIPTION
**Pull request by ahdezma**
It would be ideal if we could have a runnable example in the Apio Architect Repository, instead of relying on [Docker containers](https://github.com/liferay/com-liferay-apio-architect/blob/1729ff3e496ea8fb0b62c3aed3f5c9b31f82a894/build-ext.gradle#L179) for this.

This could be achieved by using the [Bnd Gradle Plugin](https://github.com/bndtools/bnd/tree/master/biz.aQute.bnd.gradle).

### CODE REVIEW

**Comment by nhpatt** [comment]
rebase needed, it just dependencies :P

File: apio-architect-sample/src/test/java/com/liferay/apio/architect/sample/internal/converter/PersonConverterTest.java [hunk 1]
```diff
@@ -35,6 +39,8 @@
 
 	@Test
 	public void testToPerson() {
+		RuntimeDelegate.setInstance(new RuntimeDelegateImpl());
```
Comment (victorg1991) [inline apio-architect-sample/src/test/java/com/liferay/apio/architect/sample/internal/converter/PersonConverterTest.java]: What is this?

File: apio-architect-sample/sample.bndrun [hunk 2]
```diff
@@ -1,3 +1,5 @@
+-include: -local.bnd
+
```
Comment (victorg1991) [inline apio-architect-sample/sample.bndrun]: Do I have to create another bnd and add `-runvm: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005"` if I want to enable debugging?

File: apio-architect-sample/src/test/java/com/liferay/apio/architect/sample/internal/converter/PersonConverterTest.java (hunk 1 above)
Comment (nhpatt) [reply apio-architect-sample/src/test/java/com/liferay/apio/architect/sample/internal/converter/PersonConverterTest.java]: URLCreator needs an implementation of JAX-RS (why? it doesn't make sense) and the tests aren't able to see the jar (classloader issues?). That way we set the implementation directly.

Comment (victorg1991) [reply apio-architect-sample/src/test/java/com/liferay/apio/architect/sample/internal/converter/PersonConverterTest.java]: I just show the `RuntimeDelegate.getInstance().createUriBuilder();` in the UriBuilder method 😂 . Thanks!

File: apio-architect-sample/sample.bndrun (hunk 2 above)
Comment (ahdezma) [reply apio-architect-sample/sample.bndrun]: yep 😸

**Comment by ahdezma** [comment]
Closing this and sending another rebased PR

**Pull request closed without merging**